tests/src/core/geometry/testqgsgeometry.cpp

```diff
@@ -2147,17 +2147,17 @@ void TestQgsGeometry::orientedMinimumBoundingBox()
   QVERIFY( result.isEmpty() );
 
   // oriented rectangle
-  geomTest = QgsGeometry::fromWkt( QStringLiteral( " Polygon(0 0, 5 5, -2.07106781186547462 12.07106781186547551, -7.07106781186547462 7.07106781186547551, 0 0) " ) );
+  geomTest = QgsGeometry::fromWkt( QStringLiteral( " Polygon((0 0, 5 5, -2.07106781186547462 12.07106781186547551, -7.07106781186547462 7.07106781186547551, 0 0)) " ) );
   result = geomTest.orientedMinimumBoundingBox( );
-  QgsPolygonXY geomXY, resultXY;
-  geomXY = geomTest.asPolygon();
-  resultXY = result.asPolygon();
+  QgsPolygonXY resultXY = result.asPolygon();
+  QgsPolygonXY expectedGeomXY = QgsGeometry::fromWkt( QStringLiteral( " Polygon((-7.07106781186547462 7.07106781186547551, 0 0, 5 5, -2.07106781186547462 12.07106781186547551, -7.07106781186547462 7.07106781186547551)) " ) ).asPolygon();
```

Comment:
It'd be cleaner here to replace this whole comparison with a `QCOMPARE( result.asWkt(3), QStringLiteral("....") )` check.

Reply:
This is much better indeed. Done. Thanks.